app/controllers/fhir/r4/api_controller.rb

```diff
@@ -5,41 +5,23 @@
 class Fhir::R4::ApiController < ApplicationApiController
```

Comment:
Looks like this has now shortened `ApiController` enough that the rubocop override for class length is no longer needed, so rubocop wants you to get rid of that.

Reply:
Ohh nice, that's the goal. Although if the length limit is 1000 I doubt we will last very long. But I will update.

Reply:
Alright, https://github.com/SaraAlert/SaraAlert/pull/843/commits/479ca92f8c45792beea67f8effae13f796cec701 should fix that